lib/rouge/lexers/json.rb

```diff
@@ -12,58 +12,66 @@ class JSON < RegexLexer
                 'application/hal+json', 'application/problem+json',
                 'application/schema+json'
 
-      state :root do
+      state :whitespace do
         rule %r/\s+/, Text::Whitespace
+      end
+
+      state :root do
+        mixin :whitespace
         rule %r/{/, Punctuation, :object
         rule %r/\[/, Punctuation, :array
+
+        rule(%r//) { push :value }
       end
 
       state :object do
-        rule %r/\s+/, Text::Whitespace
-        rule %r/"/, Str::Double, :name
+        mixin :whitespace
+        rule %r/"/, Name::Label, :name
         rule %r/:/, Punctuation, :value
         rule %r/,/, Punctuation
         rule %r/}/, Punctuation, :pop!
       end
-
+      
```

Comment:
@pyrmont You introduced whitespace here.
(*RuboCop can help you detect such human errors*)

Reply:
Alas, not when using GitHub's web-based code editor.

Reply:
True but Travis will fail when RuboCop detects the offense. :wink: